packages/desktop/src/renderer/components/widgets/aggressiveWarningModal/AggressiveWarningModal.component.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import Modal from '../../ui/Modal/Modal'
+import { Button, Grid, Typography } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import WarnIcon from '../../../static/images/exclamationMark.svg'
+
+const PREFIX = 'AggressiveWarningModalComponent-'
+
+const classes = {
+  bodyText: `${PREFIX}bodyText`,
+  button: `${PREFIX}button`,
+  image: `${PREFIX}image`,
+}
+
+const StyledGrid = styled(Grid)(({ theme }) => ({
+  [`& .${classes.bodyText}`]: {
+    textAlign: 'center',
+    width: '65%',
+    margin: '24px 0 4px',
+  },
+  [`& .${classes.image}`]: {
+    width: '70px',
+    height: '70px',
+    margin: '30px 0 24px',
+  },
+  [`& .${classes.button}`]: {
+    marginTop: 16,
+    textTransform: 'none',
+    padding: '0 24px',
+    height: 40,
+    borderRadius: '8px',
+    color: theme.palette.colors.white,
+    backgroundColor: theme.palette.colors.quietBlue,
+    '&:hover': {
+      opacity: 0.7,
+      backgroundColor: theme.palette.colors.quietBlue,
+    },
+  },
+}))
+
+export interface AggressiveWarningModalComponentProps {
+  communityName: string
+  leaveCommunity: () => void
+  open: boolean
+  handleClose: () => void
+}
+
+const AggressiveWarningModalComponent: React.FC<AggressiveWarningModalComponentProps> = ({
```

Comment:
What's the difference between AggresiveWarningModal, AggresiveWarningContainer and AggresiveWarningComponent? I think so far we've been using a little bit different template for modals